packages/react/src/table/docs/overview.mdx

```diff
@@ -435,6 +435,86 @@ A Table must have some form of caption or heading to describe the data it contai
 </Stack>
 ```
 
+## ‘colSpan’ and ‘rowSpan’
+
+Some tables require cells which represent multiple rows or columns. Instead of repeating the spanning data each time, you can use the `colSpan` and `rowSpan` props to specify how a specific cell spans multiple columns and rows, respectively.
+
+```jsx live
+<TableWrapper>
+	<Table>
+		<TableCaption>Items sold December 2023</TableCaption>
+		<TableHead>
+			<TableRow>
+				<TableHeader colSpan="2"></TableHeader>
```

Comment:
Is it OK to have an empty table heaader?

Reply:
Changed to a TD as discussed